bodo/pandas/utils.py

```diff
@@ -900,9 +900,18 @@ def count_plan(self):
         "LogicalAggregate",
         count_star_schema,
         self._plan,
-        0,
-        0,
-        [LazyPlan("FunctionExpression", count_star_schema, "count_star()")],
+        [],
+        [
+            LazyPlan(
+                "AggregateExpression",
+                count_star_schema,
+                self._plan,
+                "count_star",
```

Comment:
Consider changing the column name to remove () since I don't think you are using that with your new aggregate impl.

Reply:
That is above on line 898.

Reply:
Done.